### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by riveramj**
brace yourself- this PR actually updates an open source readme file.

Crazy i know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ You can include Formality directly as a dependency in your `build.sbt`:
 
 ```
 libraryDependencies ++= Seq(
-  "com.hacklanta" %% "lift-formality" % "0.4.0-SNAPSHOT"
+  "com.hacklanta" %% "lift-formality" % "1.0.0"
```
Comment (Shadowfiend) [inline README.md:24]: :tada:

**Comment by Shadowfiend** [comment]
👍

**Comment by farmdawgnation** [comment]
> brace yourself- this PR actually updates an open source readme file.

DAFUQ IS THAT ALLOWED?

**Comment by riveramj** [comment]
>DAFUQ IS THAT ALLOWED?

oh crap. i didnt even think about checking the rule book.

**Pull request merged**